src/components/HeaderForm.js

```diff
@@ -190,7 +194,7 @@ function viewTargetsEqual(currViewTarget, nextViewTarget) {
     return false;
   }
 
-  if (currViewTarget.simplify !== nextViewTarget.simplify) {
+  if (currViewTarget.removeSequences !== nextViewTarget.removeSequences) {
```

Comment:
I think we also still need to check if the `simplify` fields differ, not *just* the new `removeSequences` field.